imaging-hub.data-commons.org/metadata/aggregate_config.json

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
oops i think we need to remove this too?

Reply:
the whole file

Reply:
Removed!